File: python/7 kyu/387.py
# https://www.codewars.com/kata/58bcdc65f6d3b11fce000045/train/python

# Passed

import math

def mn_lcm(m, n):
    m, n = sorted([m, n])
    least_common_multiple = math.lcm(*list(range(m, n + 1)))
    return least_common_multiple

output = mn_lcm(1,5)
print(output)

File: nodejs/7 kyu/387.js

// https://www.codewars.com/kata/57bec3bc5640aa5f3f00003e/train/javascript

// Passed

function generateCurrencyMatrix(currency) {
  let currenciesInDescendingStrength = [
    "EUR",
    "GBP",
    "AUD",
    "NZD",
    "USD",
    "CAD",
    "CHF",
    "JPY",
  ];

  let isStrongerCurrency = false;
  let result = [];
  for (currencyToCompare of currenciesInDescendingStrength) {
    if (currency === currencyToCompare) {
      isStrongerCurrency = true;
      continue;
    }

    let currencyPair = isStrongerCurrency
      ? `${currency}${currencyToCompare}`
      : `${currencyToCompare}${currency}`;
    result.push(currencyPair);
  }

  return result;
}

const output = generateCurrencyMatrix("GBP");
console.log(output);